contracts/BnomialNFT.sol

```diff
@@ -26,11 +27,19 @@ contract BnomialNFT is ERC721, Ownable {
     constructor() ERC721("Bnomial Achievement Badge", "BNOMIAL") {}
 
     /**
-     * @dev Defines the base URI for the tokens
+     * @dev Get the base URI for the tokens
      * @return string representing the base URI
      */
-    function _baseURI() internal pure override returns (string memory) {
-        return "ipfs://QmacF9yRXkUEUHvJuCCC77JhzSLMWWJ8vFciTeVfzEoByf/";
+    function _baseURI() internal view override returns (string memory) {
```

Comment:
Considering that the compiler generates a public getter function, shouldn't we remove this one?

Reply:
Yes, you are right. I'll be removing the baseURI in the next PR so won't change it now :)